### DISCUSSION TITLE
Strategy for prioritizing variants

### DISCUSSION THREAD

**Discussion by bnouyou** [broadinstitute/seqr#2132] [General]
Hello,
I would like to have your opinion about the strategy to be used for the variant search without a gene list. 
When I do a variant search without filtering by a gene list, I obtain a large number of variants (several hundred / thousands). At this point, what is your strategy for filtering or prioritizing some variants?
Thank you in advance for your help.

**Comment by hanars** [comment]
seqr comes with predefined "saved searches" which are the recommended searches  to perform. You can select them from the dropdown on the search page
